app/services/kredits.js

```diff
@@ -30,6 +30,17 @@ export default Service.extend({
 
   ethProvider: null,
   currentUserAccounts: null, // default to not having an account. this is the wen web3 is loaded.
+  currentUser: null,
+  currentUserIsContributor: computed('currentUser', function() {
+    return Ember.isPresent(this.get('currentUser'));
```

Comment:
You can add:
`import { isEmpty, isPresent } from 'ember-utils';`
and make this `return isPresent(this.get('currentUser'));`